test/e2e/hooks_test.go

```diff
@@ -491,12 +491,12 @@ spec:
       container:
         image: argoproj/argosay:v2
         command: ["/bin/sh", "-c"]
-        args: ["/bin/sleep 1; /argosay"]
+        args: ["/bin/sleep 5; /argosay"]
     - name: argosay-sleep-2seconds
       container:
         image: argoproj/argosay:v2
         command: ["/bin/sh", "-c"]
-        args: ["/bin/sleep 2; /argosay"]
+        args: ["/bin/sleep 1; /argosay"]
```

Comment:
how about this, is reduced to 1s enough?
If so, could you update the step name to 1second

Reply:
Oh, I feel like this sleep is of no use, so I turned it down. What works is the previous sleep. I forgot to change the name. Thanks

Reply:
Returned to 2 because I saw many other similar ones using 2.